boards/riscv/litex_vexriscv/litex_vexriscv.dts

```diff
@@ -51,6 +51,13 @@
 &i2c0 {
 	status = "okay";
 	label = "I2C_0";
+
+	ov2640@30 {
+		compatible = "ovti,ov2640";
+		reg = <0x30>;
+		label = "OV2640";
+		status = "okay";
```

Comment:
Is this sensor always connected to that board? is it an 'official' expension board? or a custom setup?

Reply:
It's custom setup, I used [this](https://www.waveshare.com/ov2640-camera-board.htm) camera as mentioned before.

Reply:
Ok, so you can not enable this specific device in the mainline dts, since it's specific to your setup. You can either remove that change from the series and keep your change downstream, or alternatively I think you can add the node but with `status = "disabled"` by default with proper documentation to explain how replicating your setup. First solution is probably simpler ;-)

Reply:
Thanks, I guess I'll get rid of that part then :)
I removed that commit.